torch/utils/cpp_extension.py

```diff
@@ -1482,4 +1568,4 @@ def _join_cuda_home(*paths):
 
 
 def _is_cuda_file(path):
-    return os.path.splitext(path)[1] in ['.cu', '.cuh']
+    return os.path.splitext(path)[1] in ['.cu', '.cuh', '.hip']
```

Comment:
It would be nice if we had some sort of

```
valid_files = ['.cu', '.cuh']
if rocm_extension:
  valid_files.append('.hip')
```
so that some eager dev doesn't come along and delete the '.hip' from here accidentally

Reply:
done